### PR TITLE
add manylinux build script

### PR DESCRIPTION
**Pull request by micmelesse**

### CODE REVIEW

File: tensorflow/tools/tf_sig_build_dockerfiles/manylinux_build.sh [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+# build wheel
+cd $BUILD_DIR/tensorflow && yes "" | TF_NEED_ROCM=1 ROCM_TOOLKIT_PATH=${ROCM_INSTALL_DIR} PYTHON_BIN_PATH=${PYTHON_BIN_PATH} ./configure &&
+    bazel build --config=opt --action_env TF_ROCM_GCC=1 --config=rocm //tensorflow/tools/pip_package:build_pip_package --verbose_failures &&
+    bazel-bin/tensorflow/tools/pip_package/build_pip_package $TF_PKG_LOC --rocm --project_name tensorflow_rocm
+
+# check that wheels are manylinux compatiable
+TF_WHEEL=$(ls -Art $TF_PKG_LOC/tensorflow*.whl | tail -n 1)
+echo "Checking $TF_WHEEL..."
+pip3 install auditwheel
+time auditwheel repair --plat manylinux2014_x86_64 "$TF_WHEEL" --wheel-dir $TF_PKG_LOC 2>&1 | tee ${TF_WHEEL}_check.txt
```
Comment (jayfurmanek) [inline tensorflow/tools/tf_sig_build_dockerfiles/manylinux_build.sh]: For some reason `auditwheel` repair is breaking or removing ROCm support. Let's change this to a simple filename swap. We'll just need to swap `linux` to `manylinux2014` like

```
NEW_TF_WHEEL=${TF_WHEEL/linux/"manylinux2014"}
echo "rename whl to $NEW_TF_WHEEL"
mv $TF_WHEEL $NEW_TF_WHEEL
```

Comment (micmelesse) [reply tensorflow/tools/tf_sig_build_dockerfiles/manylinux_build.sh]: Done.

Comment (micmelesse) [reply tensorflow/tools/tf_sig_build_dockerfiles/manylinux_build.sh]: Will see how ci reacts and then cleanup

**Comment by micmelesse** [comment]
@jayfurmanek Shall this be merged or closed?

**Comment by jayfurmanek** [comment]
oh - let's close this. Most of this was merge elsewhere.

**Pull request closed without merging**